components/CreateForm/PreviewActionButtons.tsx

```
@@ -0,0 +1,58 @@
+import { useRef } from 'react';
+
+interface PreviewActionButtonsProps {
+  isUploading: boolean;
+  onUploadClick: () => void;
+  onDoneClick: () => void;
+  onFileChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const PreviewActionButtons = ({
+  isUploading,
+  onUploadClick,
```

ESLint

[error] 'onUploadClick' is defined but never used.

(@typescript-eslint/no-unused-vars)